### ISSUE TITLE
Invalid escape sequences in strings

### ISSUE THREAD

**Issue by QuLogic** [pysal/mapclassify#57]
There are some warnings when bytecompiling files in Python 3.8; this may eventually stop working in newer Python versions.
```
mapclassify-2.2.0/mapclassify/classifiers.py:568: DeprecationWarning: invalid escape sequence \l
mapclassify-2.2.0/mapclassify/classifiers.py:2593: DeprecationWarning: invalid escape sequence \s
```

**Comment by sjsrey** [comment]
Thanks for spotting this.

**Comment by sjsrey** [comment]
https://stackoverflow.com/questions/50504500/deprecationwarning-invalid-escape-sequence-what-to-use-instead-of-d

**Comment by QuLogic** [comment]
You can mark as raw strings, or escape the backslash (just double them), depending on how well the string reads.

**Comment by sjsrey** [comment]
Fixed with #58.
